changelog/v0.31.3/conditional_crd_rendering.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
can you make this a BREAKING_CHANGE and bump the minor version (v0.32.0) since it has function signature changes?

Reply:
done